src/main/java/org/calrissian/flowmix/api/aggregator/LongSumAggregator.java

```diff
@@ -32,6 +32,7 @@
 import static java.lang.System.currentTimeMillis;
 import static java.util.UUID.randomUUID;
 
+@Deprecated
 public class LongSumAggregator implements Aggregator {
```

Comment:
Totally agree, I'm going to think about something, BUT you could always use the brand new FunctionAggregator for your own 'short' or 'byte' implementation :)